checkov/common/util/secrets_omitter.py

```diff
@@ -80,9 +80,11 @@ def omit(self) -> SecretsOmitterStatus:
         files_with_secrets: set[str] = {secret_check.get("file_path", "") for secret_check in self._secret_check()}
         for check in self._non_secret_check():
             check_file_path = check.file_path
+            check_abs_file_path = check.file_abs_path
             check_line_range = check.file_line_range
 
-            if check_file_path not in files_with_secrets or not check_line_range or None in check_line_range:
+            if (check_file_path not in files_with_secrets and check_abs_file_path not in files_with_secrets)\
```

Comment:
feels weird to need to do both file path checks, but we probably can't know, which one is stored in `files_with_secrets` 😢

Reply:
Yeah since it's up to each Runner to decide how the record is stored I guess that in this case, we are better off checking both fields instead of trying to align everything :/